docs/QuickStart.md

```diff
@@ -48,126 +48,13 @@ ALTER TABLE ['{table_name}'] ADD CONSTRAINT PKey PRIMARY KEY CLUSTERED (['{prima
 
 Now you will need a Function App to add the binding to. If you have one created already you can skip this step.
```

Comment:
This sentence doesn't make sense anymore. And the flow is messed up now - users will need to : 

1. Start here to configure their table
2. Somehow figure out that they need to go into the language specific guides to create their function app
3. Come back here to finish configuring it

I'm not sure I like the idea of duplicating all the content in this file in each language guide - please see if you can come up with something better (and make sure you're walking through the guide yourself to verify things make sense and flow well)

Reply:
I think I will keep the stuff in the Quickstart for now as each language has a section that indicates users to go to the quickstart guide to go setup function app further. I just believe that not having the host.json preview bundle in the language is also not a great user experience. Duplicating information is better than having a confused user.

Reply:
The bottom of quickstart still references having the user go to the language specific guides. So if you think it makes sense like that then let's fix everything up to follow logically with appropriate links at each stage. I would also suggest we rename this from `QuickStart.md` to something more general like `GeneralSetup.md` since QuickStart to me implies the user should be starting there.

Reply:
reverted the fix for now. I believe having the information in both places is fine as we include `If you have one created already you can skip this step.` so that if a user started in language-specific doc or general startup they could understand that they would not need it.